packages/webui/src/components/Authorization.tsx

```diff
@@ -105,7 +105,7 @@ export default function Authorization({ value }: AuthorizationProps) {
         }
       })()}
       {tokenState !== TokenState.Minimal && (
-        <div className={tw('flex flex-row gap-2 bg-slate-100 px-4 pt-4')}>
+        <div className={tw('flex flex-row gap-2 bg-gray-200 px-4 pt-4')}>
```

Comment:
On expansion, the display mode and the collapse buttons don't look great.  Not sure whether there is a plan to address this in another PR?

![image](https://github.com/FormidableLabs/envy/assets/17857833/31ddb3c7-bc96-4983-9e50-b0126f48cb84)